src/screens/UserPortal/People/People.module.css

```diff
@@ -48,13 +48,13 @@
 }
 
 .colorLight {
-  background-color: #f5f5f5;
+  /* background-color: #f5f5f5; */
```

Comment:
Remove the commented.code. Maybe you can remove entire colorlight selector  to make changes clean

Reply:
i'm so sorry for this, you can check the latest commit